modules/consensus/consensusdb.go

```diff
@@ -52,14 +49,17 @@ var (
 	// inconsistencies within the database have been detected.
 	Consistency = []byte("Consistency")
 
-	// SiacoinOutputs is a database bucket that contains all of the unspent
-	// siacoin outputs.
-	SiacoinOutputs = []byte("SiacoinOutputs")
-
 	// FileContracts is a database bucket that contains all of the open file
 	// contracts.
 	FileContracts = []byte("FileContracts")
 
+	prefixDSCO = []byte("dsco_")
+	prefixFCEX = []byte("fcex_")
```

Comment:
hmm. Might be nitpicking, but we probably want to keep exported and unexported variables separate (i.e. stick these back at the top of the `var` block)

Reply:
I moved prefixDSCO and prefixFCEX back at the top and added the extra `var` block to separate them (otherwise `sorted_consts` complains).